x-pack/plugins/security_solution/server/endpoint/schemas/artifacts/lists.ts

```diff
@@ -20,6 +20,9 @@ export const translatedEntryMatchAny = t.exact(
 );
 export type TranslatedEntryMatchAny = t.TypeOf<typeof translatedEntryMatchAny>;
 
+export const translatedEntryMatchAnyMatcher = translatedEntryMatchAny.type.props.type;
```

Comment:
Another way to do this would be to define a type like `MatchAny`:
```
    t.keyof({
      exact_cased_any: null,
      exact_caseless_any: null,
    })
```
Then use this new type in both `translatedEntryMatchAny` and `translatedEntryMatchAnyMatcher`